src/nhamiltonian/nHamiltonian.h

```diff
@@ -631,6 +631,10 @@ MATRIX ETHD3_forces(const MATRIX& q, const MATRIX& p, const MATRIX& invM, double
 MATRIX ETHD3_friction(const MATRIX& q, const MATRIX& p, const MATRIX& invM, double alp, double bet); // dH_{ETHD3}/dP
 
 
+///< In nHamiltonian_compute_KCRPMD.cpp
+double KCRPMD_effective_potential(vector<double>& auxiliary_y);
```

Comment:
please expose this function to Python, in libdyn.cpp (see other examples) 

also - it may be better to avoid caps - e.g. "kcrpmd_effective_potential"

Reply:
So I might be mistaken here, but taking Ehrenfest as an example, I don't the the Ehrenfest_energy_dia() function within libdyn.cpp, instead I'm seeing that it's called in the src/dyn/Energy_and_Forces.cpp function potential_energies(), and it's this potential_energies() function that gets exposed to Python in libdyn.cpp. Following this example shouldn't kcrpmd_effective_potential() just be added to the definition of potential_energies() as an additional force_method option?